project-overview.md

```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
 ### Tokens
 
 Many real world attestation solutions and potential extrapolations of their use have been considered to ensure that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+* EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
+* ARM Confidential Computing Architecture (CCA) Token and its variant [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
```

Comment:
_variant_ is probably not the right word (there's no variation), _composition_ maybe more correct

Reply:
A variant is another version of something. This is correct usage, as Parsec tpm is a variant of base CCA token

Reply:
hmm, the CCA token is reused as-is by Parsec CCA, so there is absolutely no variation.  It is a composition of CCA and another piece that's stacked on top.

Reply:
ok, without being hung up on the term, I have made the change!